### ISSUE TITLE
Beginer at running a server

### ISSUE THREAD

**Issue by Maazda** [splewis/csgo-practice-mode#52]
Hi no matter where I install the files I don't get .setup to work I have admin level "99:z" and have no clue what to do. I know this probably isn't the right place to post this but couldn't find a better solution.

to let you know what I have done, I have tried installing it in a few different places. I have the admin rights.
I am able to use sm admin as well as "BFG" warmod 

can someone explain in layman's terms?

Thanks in advance.

**Comment by splewis** [comment]
To install you should take [practicemode_1.2.0.zip](https://github.com/splewis/csgo-practice-mode/releases/tag/1.2.0), and extract to the directory called ``csgo``. There should already be an addons directory there you're merging with.

If you did it correctly, you should see practicemode.smx under ``addons/sourcemod/plugins``.

If it's there, you should debug by checking error logs in ``addons/sourcemod/logs``.

**Comment by Maazda** [comment]
Thank you worked great 👍
